### ISSUE TITLE
Different controls sets / display messages for different levels

### ISSUE THREAD

**Issue by dawnho** [wearhere/spacecontrol#21]
We would like to have different sets of command sets, and different display text for those commands at different levels.

**Comment by wearhere** [comment]
Can you provide an example? Would this be swapping out which controls you ask users to manipulate, or describing using those controls in different ways?

**Comment by svetlichnaya** [comment]
@wearhere yes this was an idea from last night. Example: you have a slider (analog potentiometer) that picks a color of the rainbow. At level 1, you tell the user "Set color to [red/orange/green]". At level 2, you instead say "Banana color" or even just "Strawberries" and players have to collectively figure out to guess the color of the fruit and then set the color switch. At level 3, you could imagine giving more challenging/abstract clues for which the answer is a color (RGB values :P). This lets us include a puzzle-solving component if we want :)

**Comment by wearhere** [comment]
![](http://gifrific.com/wp-content/uploads/2012/08/mindblown-kramer.gif)

**Comment by wearhere** [comment]
That's great! That's like in Spaceteam where some levels label the controls with pictographs.

**Comment by wearhere** [comment]
Ok here's the schema change I'm envisioning for this:

```json
'actions': {
  'nothing': '',
  'nipple': 'Octo bite raven girl nipple!',
  'mouth': 'Octo kiss raven girl mouth!'
}
```

becomes

```json
'actions': {
  'nothing': '',
  'nipple': 'Octo bite raven girl nipple!',
  'mouth': {
    '0': 'Octo kiss raven girl mouth!'
    '1': 'Octo frenches raven!'
    '3': 'Octo just wants some of raven's extra time and her…'
  }
}
```

That is to say, the right-hand value of each state can be expanded from a single action into a dictionary mapping difficulty levels to actions.

The difficulty levels should be numbers `'0'` through however many levels you care to provide _as strings_.

It's ok for different controls to support different numbers of difficulty levels or even just a single difficulty level i.e. the old single-action form (like `'nipple'`'s action above), the server will figure everything out.

No protocol changes.

**Comment by wearhere** [comment]
The above scheme is designed to support changing the actions every level. If we instead want to support changing the actions per _range_ of levels, then it might make sense to adopt min/maxDifficulty keys at the control level: https://cl.ly/2x2N3i2b3R0y h/t @dawnho

**Comment by wearhere** [comment]
Controls would still require unique IDs when if they weren't simultaneously active.

**Comment by wearhere** [comment]
*even if
